packages/itwin/map-layers/src/test/useInterval.test.tsx

```diff
@@ -2,40 +2,30 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import { expect } from "chai";
-import sinon from "sinon";
 import { renderHook } from "@testing-library/react";
-import { useInterval } from "../ui/hooks/useInterval";
+import { useInterval } from "../../src/ui/hooks/useInterval";
 
 describe("useInterval", () => {
-  let clock: sinon.SinonFakeTimers;
-
-  beforeEach(() => {
-    clock = sinon.useFakeTimers();
-  });
-
-  afterEach(() => {
-    clock.restore();
-  });
-
   it("should call interval's callback when timeout is reached", () => {
-    const spy = sinon.spy();
+    vi.useFakeTimers();
```

Comment:
after using fake timers, make sure you reset them ([ref](https://vitest.dev/api/vi.html#vi-userealtimers)). If you have two tests that will utilize these timers, I'd call these in a beforeEach/afterEach test hook

Reply:
oh, i see that you've implemented them in the setup oops. But should that be in there? seems like fake timers are used in only a few places